packages/unmock-jsdom/src/backend/index.ts

```diff
@@ -117,6 +116,28 @@ export default class JSDomBackend implements IBackend {
           return XMLHttpRequestSetRequestHeader.apply(this, [name, value]);
         }
       };
+      const doEndReporting = (fromCache: boolean, responseBody: string, responseHeaders: any) =>
+        endReporter(
+          "HASH", // fix!
```

Comment:
Was this meant to be fixed before the PR?

Reply:
Ah crap, yeah :-(

Reply:
The problem is in reporting to the CLI. Everything works, but it is printed as `"HASH"`, which is why no test caught it. I will write tests for `endReporter` at some point.